lib/linguist/languages.yml

```
@@ -3053,6 +3053,7 @@ KiCad Schematic:
   - eeschema schematic
   extensions:
   - ".sch"
+  - ".kicad_sch"
```

I'd make this the primary extension for KiCad schematics, as it appears the `.sch` extension is [considered](https://docs.kicad.org/master/en/kicad/kicad.html#kicad_files_and_folders) deprecated:

> `*.sch` | Legacy (KiCad 5.x and earlier ) schematic file. Can be read and will be converted to a `.kicad_sch` file on write.
> -- | :--

```suggestion
  - ".kicad_sch"
  - ".sch"
```